src/common/snippets/src/op/serialization_node.cpp

```diff
@@ -40,22 +40,48 @@ std::shared_ptr<Node> SerializationNode::clone_with_new_inputs(const OutputVecto
 }
 
 bool SerializationNode::visit_attributes(AttributeVisitor &visitor) {
+    auto is_planar_layout = [](const std::vector<size_t>& layout) {
+        for (size_t i = 0; i < layout.size(); ++i)
+            if (layout[i] != i) return false;
+        return true;
+    };
+
     std::vector<size_t> in_regs, out_regs;
     std::vector<std::string> in_reg_types, out_reg_types;
-    std::vector<std::pair<std::string, std::vector<size_t>>> shapes;
+    std::vector<std::pair<std::string, VectorDims>> shapes;
+    std::vector<std::pair<std::string, VectorDims>> subtensors;
+    std::vector<std::pair<std::string, std::vector<size_t>>> layouts;
     for (size_t i = 0; i < m_expr->get_input_count(); i++) {
         const auto& desc = m_expr->get_input_port_descriptor(i);
-        const auto &shape = desc->get_shape();
+        const auto& shape = desc->get_shape();
         if (!shape.empty())
             shapes.emplace_back("in_shape_" + std::to_string(i), shape);
+
+        const auto& subtensor = desc->get_subtensor();
+        if (!subtensor.empty())
+            subtensors.emplace_back("in_subtensor_" + std::to_string(i), subtensor);
```

Comment:
Honestly, I thought about that. But decided to leave it so since in general it's ok 😄 
But if another developer says that it's huge, I added support of `subtensor2str` in https://github.com/openvinotoolkit/openvino/pull/24289/commits/cbfb2c7532dd37742b4cb5e042a52c82a52e91aa:
![image](https://github.com/openvinotoolkit/openvino/assets/43129309/5f9c363c-d4a8-4875-8c49-eac4f3f47569)
Thanks!

Reply:
Perfect, thank you!